canonical/charmhub.io
# Populate developer documentation section of interface

## Done

## How to QA
- Go to https://charmhub-io-1437.demos.haus/interfaces/ingress-v0
- Check that there is a populate "Developer documentation" section
- **Note**: The markdown formatting isn't quite right but will be corrected in https://warthogs.atlassian.net/browse/WD-665

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-379
